### ISSUE TITLE
Cleanup error

### ISSUE THREAD

**Issue by it-can** [Homebrew/homebrew#41925]
When I try to do ```brew cleanup``` I get this error:

```
Error: undefined method `strip' for nil:NilClass
Please report this bug:
    https://git.io/brew-troubleshooting
/usr/local/Library/Homebrew/extend/pathname.rb:396:in `abv'
/usr/local/Library/Homebrew/keg.rb:139:in `abv'
/usr/local/Library/Homebrew/cmd/cleanup.rb:96:in `cleanup_path'
/usr/local/Library/Homebrew/cmd/cleanup.rb:61:in `cleanup_keg'
/usr/local/Library/Homebrew/cmd/cleanup.rb:46:in `block in cleanup_formula'
/usr/local/Library/Homebrew/cmd/cleanup.rb:46:in `each'
/usr/local/Library/Homebrew/cmd/cleanup.rb:46:in `cleanup_formula'
/usr/local/Library/Homebrew/cmd/cleanup.rb:35:in `block in cleanup_cellar'
/usr/local/Library/Homebrew/cmd/cleanup.rb:33:in `each'
/usr/local/Library/Homebrew/cmd/cleanup.rb:33:in `cleanup_cellar'
/usr/local/Library/Homebrew/cmd/cleanup.rb:12:in `cleanup'
/usr/local/Library/brew.rb:124:in `<main>'
```

Does anybody have an idea what to do?

I am using:
- Mac OS X: 10.10.4
- Xcode latest installed (and accepted agreement)
- Homebrew 0.9.5

**Comment by bfontaine** [comment]
Could you please run `brew update` and try again?

**Comment by it-can** [comment]
nope... still the same...

**Comment by DomT4** [comment]
Can you do `cd $(brew --prefix) && git status` and see if anything is unexpectedly modified?

**Comment by it-can** [comment]
```
cd $(brew --prefix) && git status
On branch master
nothing to commit, working directory clean
```

**Comment by bfontaine** [comment]
Could you please post the output of `brew config` and `brew doctor`?

**Comment by it-can** [comment]
```
brew doctor
Your system is ready to brew.
```

```
brew config
HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/Homebrew/homebrew
HEAD: 7a19d3185c53755769e257ec01d1d93d721df850
Last commit: 5 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: 8-core 64-bit haswell
OS X: 10.10.4-x86_64
Xcode: 6.4
CLT: 6.4.0.0.1.1435007323
Clang: 6.1 build 602
X11: N/A
System Ruby: 2.0.0-p481
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /usr/bin/ruby
Java: 1.6.0_65
```

**Comment by xu-cheng** [comment]
This will be fixed by #42011

**Comment by it-can** [comment]
Yes this works! It now shows the problem directories... thanks
